modules/consensus/consensusset.go

```diff
@@ -358,3 +358,45 @@ func (cs *ConsensusSet) StorageProofSegment(fcid types.FileContractID) (index ui
 	})
 	return index, err
 }
+
+// GetConsensusChange returns the ConsensusChange that corresponds to a given ConsensusChangeID. Primarily used for API
+// calls as modules should Subscribe to the Consensus Set instead
+func (cs *ConsensusSet) GetConsensusChange(id modules.ConsensusChangeID) (change modules.ConsensusChange,
+	next modules.ConsensusChangeID, err error) {
```

Comment:
we have a convention of never splitting the function declaration across lines, no matter how long they are. :P